engine/platform/src/platform_window_null.cpp

```diff
@@ -106,6 +109,9 @@ namespace dmPlatform
         }
     }
 
+    void SetWindowPosition(HWindow window, int32_t x, int32_t y)
+    {}
+
```

Comment:
Note that this is a change for consoles as well.
Should be a small fix though, but worth remembering.

Reply:
Yes, we need null implementations on consoles too I suppose?